pkgs/applications/graphics/akira/unstable.nix

```diff
@@ -0,0 +1,57 @@
+{ stdenv, lib, fetchFromGitHub, appstream-glib, desktop-file-utils, meson, ninja
```

Comment:
one per line of the function formal arguments please.
```
{ stdenv,
, lib
, fetchFromGitHub
```

Reply:
Sorry, pesky formatter gets in the way when I don't notice. :eyes: